aspnetcore/blazor/host-and-deploy/configure-trimmer.md

```diff
@@ -42,73 +42,123 @@ For more information, see [Trimming options (.NET documentation)](/dotnet/core/d
 
 ## Failure to preserve types used by a published app
 
-Trimming may have detrimental effects for a published app leading to runtime errors. In apps that use [reflection](/dotnet/csharp/advanced-topics/reflection-and-attributes/), the IL Trimmer often can't determine the required types for runtime reflection and trims them away or trims away parameter names from methods. This can happen with complex framework types used for JS interop, JSON serialization/deserialization, and other operations.
+Trimming may have detrimental effects for a published app leading to runtime errors, even in spite of setting the [`<PublishTrimmed>` property](#configuration) to `false` in the project file. In apps that use [reflection](/dotnet/csharp/advanced-topics/reflection-and-attributes/), the IL Trimmer often can't determine the required types for runtime reflection and trims them away or trims away parameter names from methods. This can happen with complex framework types used for JS interop, JSON serialization/deserialization, and other operations.
```

Comment:
> even in spite of setting the [`<PublishTrimmed>` property](#configuration) to `false`

This doesn't sound right. Setting `PublishTrimmed` to false should disable trimming for the entire app. Does this not work? If it doesn't, I think it's a bug that we should fix, not document.

Reply:
I'll double check this in the morning and report back.

Reply:
@danroth27 ... ***Confirmed!***

<img width="1018" height="130" alt="image" src="https://github.com/user-attachments/assets/c0c5ec46-62a9-4b28-a3bc-859db5f6238f" />

<img width="1876" height="406" alt="image" src="https://github.com/user-attachments/assets/a4e875d0-2327-44d6-847b-c41c8d60409e" />

Reply:
@gaurdrex Can you create an issue for this in the dotnet/aspnetcore repo and share your project with us?

Reply:
@danroth27 ... Isn't this the same as (or at least has the elements of) https://github.com/dotnet/aspnetcore/issues/52947? I thought @javiercn was well aware of this, and I placed the content that he requested to cover it. I just want to make sure I'm opening an issue that @javiercn won't close saying that it can never be fixed.

Reply:
... and @danroth27 ... are we holding this past the RC1 release if @javiercn doesn't respond by then?

Reply:
I just asked because you were calling for his review. I'll react with a patch PR later if he has any changes.